tests/Role/ReadModel/Permissions/Doctrine/UserPermissionsReadRepositoryTest.php

```diff
@@ -46,7 +46,7 @@ public function setUp(): void
      */
     public function it_should_return_the_permissions_for_a_user_that_are_granted_by_his_roles(): void
     {
-        $userId = new StringLiteral('7D23021B-C9AA-4B64-97A5-ECA8168F4A27');
+        $userId = '7D23021B-C9AA-4B64-97A5-ECA8168F4A27';
         $roleId = new StringLiteral('7B6A161E-987B-4069-8BB2-9956B01782CB');
```

Comment:
I think all `StringLiteral` instances can be removed from this file.

Reply:
I've removed the unnecessary ones. There's a SchemaConfigurator where the constructor still relies on a StringLiteral. Refactoring that away in this PR would cause too many changes. Maybe a separate PR?

Reply:
You are right, they still need to be used for `SchemaConfigurator` but can be removed inside `it_should_return_the_permissions_for_a_user_that_are_granted_by_his_roles`